salt/utils/event.py

```diff
@@ -147,11 +147,9 @@ def get_master_event(opts, sock_dir, listen=True, io_loop=None, raise_errors=Fal
     """
     Return an event object suitable for the named transport
     """
-    # TODO: AIO core is separate from transport
-    if opts["transport"] in ("zeromq", "tcp", "detect"):
-        return MasterEvent(
-            sock_dir, opts, listen=listen, io_loop=io_loop, raise_errors=raise_errors
-        )
+    return MasterEvent(
```

Comment:
on line 109 above we specify hard-coded transport as "zeromq", but the value is never used.

Reply:
I've noticed this too.. This should be tackled in another PR. I've created this ticket https://github.com/saltstack/salt/issues/61275 and assigned it to @MKLeb